src/nlp.jl

```diff
@@ -125,12 +125,12 @@ mutable struct NLPEvaluator <: MathProgBase.AbstractNLPEvaluator
     hess_I::Vector{Int}
     hess_J::Vector{Int}
     max_chunk::Int # chunk size for which we've allocated storage
-    # timers
-    eval_f_timer::Float64
-    eval_g_timer::Float64
-    eval_grad_f_timer::Float64
-    eval_jac_g_timer::Float64
-    eval_hesslag_timer::Float64
+    # init flags
+    eval_f_init::Bool
+    eval_g_init::Bool
```

Comment:
Are any of these flags used except for eval_f_init? Please remove them if not.

Reply:
Apparently they all get set, but whether they actually get used for anything I don't know (probably not within JuMP).  Perhaps best to just leave it as this is only 0.18.

Reply:
And when I say "they all get set" I mean there are other functions that set them, not just when they are initialized in the constructors.